packages/commonwealth/server/routes/getNewProfile.ts

```diff
@@ -108,13 +111,25 @@ const getNewProfile = async (
     },
   });
 
+  const profileTags = await models.ProfileTags.findAll({
+    where: {
```

Comment:
No code changes required here, but instead in the schema.

Since we are querying the ProfileTags by profileId, and we expect to have a lot of ProfileTags in the database, we should add an index on ProfileTags so that this operations becomes O(logn) instead of O(n)